treeherder/intermittents_commenter/commenter.py

```diff
@@ -459,9 +497,10 @@ def build_bug_map(self, bugs, option_collection_map):
                 data_table = bug_infos.data_table
                 platform_and_build_data = data_table.get(platform_and_build, {})
                 data_table[platform_and_build] = platform_and_build_data
-                data_table[platform_and_build][test_variant] = (
-                    platform_and_build_data.get(test_variant, 0) + 1
-                )
+                if test_variant in data_table[platform_and_build]:
+                    data_table[platform_and_build][test_variant]["count"] += 1
```

Comment:
what scenario would the count increase?  is this safe to know we are counting all of the run_count properly?